source/kubeconfig_aws001-preprod-dev-eks

```diff
@@ -4,8 +4,8 @@ kind: Config
 
 clusters:
 - cluster:
-    server: https://72A9C8FA6B9F95A1A9543EAD0B148B2C.gr7.eu-west-1.eks.amazonaws.com
-    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUM1ekNDQWMrZ0F3SUJBZ0lCQURBTkJna3Foa2lHOXcwQkFRc0ZBREFWTVJNd0VRWURWUVFERXdwcmRXSmwKY201bGRHVnpNQjRYRFRJeE1EY3dOREEzTWpRek5Gb1hEVE14TURjd01qQTNNalF6TkZvd0ZURVRNQkVHQTFVRQpBeE1LYTNWaVpYSnVaWFJsY3pDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBS3NECmwrYi9OaU56Qnkrd3A5R0pRUmtPZG5OdE5UK0VWRHFYOXBvQnNnKy9nTHhicGRMZUhNMVBTR0ZBVTNNTEJRVFoKd3FaVi9IN2NGQVFqM0FBd2orYStYOXRhZUhOVXpkc1R6RUhmdEpVQXVRY29DUkNxSFU5aTVid3h5VUhHNTRvWAp1WFVXRU1jOFpkNDZYelJ6T282K09aMjVmSFU1Z3FlTUMwOEE0M29ZVXVqSHdkZVRoOGlaUUlobUljQWQzbU5YClJOanhVMTkvR00yZ3p4UGFEMUw3MHc0S0xBbTk4bDFZVGN1OW45TnJ3dzJxS3lWNnBiTWV3ck5NZVVkbTdMeUMKdUFXZjhUYTVBendSb0pyOXltRlUwQVA5YXhiTm9ZYkRraGlqdzJKcVphYTkzcEtKSks4M3Jxa2hDbFVLcFJCZgpLaThOSTZyb2cvSkozUXBncDNVQ0F3RUFBYU5DTUVBd0RnWURWUjBQQVFIL0JBUURBZ0trTUE4R0ExVWRFd0VCCi93UUZNQU1CQWY4d0hRWURWUjBPQkJZRUZLaTJWdWppTjZXMEhzZFNCVkVaNkNuWHNFblhNQTBHQ1NxR1NJYjMKRFFFQkN3VUFBNElCQVFBaDAzVXBvN3lTTHd1dGdCTlgxTTF6dmtpdDcwMnIrRmVRcytJL2NMdEM4RkpzRkdBSgpUQ1pWa0tSOGU4ZmJBU25lalkwTzdMU0FZampkWmFJbW1pYnB0eUhhWUpOZTJqODEweTE2Nk45TUVmWC9tYmxpCmJzUjh0V3JNVGM2NitxbGFYQzJvYzJieTQ1QUMrZHh1RGxhZ0JXTU5yY1h0SjdSaUFSS3FNVVpKV013N3h2UncKaGNRc1REWURFSUljVnJCeTREeGRLV1hGN2xJL04vMzJDcjE1amdnWDNMaWk3YWNpSWFQYUMxVS9veVJ0NXhneQo4Q2NhbktrWTZaK0N1STdaTUZHenRadTZ0RWkxSGJvelM2RnlkUzA5OWtoQnlEc0ZtNTZ2aWJ6MExJazAydDNBCmdqVG4yZ2NKVkRUaGxScUlZVDdiOHJzZGJwRUhoY0d2L0l3LwotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
+    server: https://E2A6FA7A4933FCCBF024434B6B83F8C8.gr7.eu-west-1.eks.amazonaws.com
```

Comment:
@UlaganathanNamachivayam  Could you please remove this file from this PR? We don't want to save this file